The time required for servicing transmissions is normally distributed with µ = 45 minutes and σ = 8 minutes. The service manager plans to have work begin on the transmission of a customer’s car 10 minutes after the car is dropped off and the customer is told that the car will be ready within 1 hour from drop-off. What is the probability that the service manager cannot meet his commitment?


In [1]:
from scipy import stats
from scipy.stats import norm

Let 'X' be the amount of time it takes to complete the repair on a customer's car. So the Probability of completing the car repair with 1 hr of dropping the car and starting the work after 10 mins of car drop (60 mins - 10 mins) is P(X<=50 mins) So, P(X>50) = 1-P(X<=50) i.e. cannot meet his commitment

In [2]:
# Find Z-Scores at X=50; Z = (X - µ) / σ 
Z=(50-45)/8
Z

0.625

In [3]:
# Find probability P(X>50) = 1-stats.norm.cdf(abs(z_score))
1-stats.norm.cdf(abs(0.625))

0.26598552904870054

In [4]:
# OR Find probability P(X<=50); p_value=stats.norm.cdf(abs(z_score))  
p_value=stats.norm.cdf(abs(0.625)) 
p_value

0.7340144709512995

In [5]:
# P(X>50) = 1 - P(X<=50)
1-0.734

0.266

We have a normal distribution with   = 45 and   = 8.0. Let X  be the amount of time it takes to complete the repair on a customer's car. To finish in one hour you must have X ≤ 50 so the question is to find Pr(X > 50).Pr(X > 50) = 1 -Pr(X ≤ 50).Z = (X -)/  = (X -45)/8.0Thus the question can be answered by using the normal table to findPr(X ≤ 50) = Pr(Z ≤ (50 -45)/8.0) = Pr(Z ≤ 0.625)=73.4%Probability that the service manager will not meet his demand will be = 100-73.4 = 26.6% or 0.2676